org.omg.sysml.xpect.tests/src/org/omg/sysml/xpect/tests/validation/invalid/MetadataUsage_Invalid.sysml.xt

```diff
@@ -49,31 +49,29 @@ XPECT_SETUP org.omg.sysml.xpect.tests.validation.invalid.SysMLTests
 END_SETUP 
 */
 package Test {
-	import ScalarValues::*;
```

Comment:
I removed the imports and use qualified names, because the updated implementation of the (parsable but invalid) filters in this test ended up (correctly!) filtering out the imports anyway.